addons/website_slides/models/slide_slide.py

```diff
@@ -706,7 +706,8 @@ def copy_data(self, default=None):
         """Sets the sequence to zero so that it always lands at the beginning
         of the newly selected course as an uncategorized slide"""
         default = dict(default or {})
-        default['sequence'] = 0
+        if 'slide.channel' not in self._context.get('__copy_data_seen', {}) and (default is {} or 'sequence' not in default):
```

Comment:
but why?

Reply:
I meant it like this sorry, we want to check for when we are coming from duplicating a whole course, and when already having a sequence in default to not set the default to 0, right?